### ISSUE TITLE
admin grid

### ISSUE THREAD

**Issue by Zydol** [DemacMedia/Magento-Multi-Location-Inventory#65]
Hello, how can I add warehauses with quantity of product to admin/product/grid in QTY column? 
Regards
Tom

**Comment by hakanvar** [comment]
Hi,
/app/design/adminhtml/default/default/template/sales/items/column/qty.phtml

Copy and Paste attached code to your file and you 
[qty.txt](https://github.com/DemacMedia/Magento-Multi-Location-Inventory/files/2588967/qty.txt)
will see stocks in order.,

**Comment by Zydol** [comment]
this show just one warehouse and stock in sales grid, I need all warehouses and in products grid

**Comment by hakanvar** [comment]
This shows all warehouses in stock in sales. If there is stock, it shows warehouse name and stock qty. If there is out of stock, wont show name and qty.

**Comment by Zydol** [comment]
yeah, but show warehose and stock from which client bought product. I need show all stock and warehouses on product grid in QTY column. Any idea?
